From 7437abf6143e7836178f832f5620c23011691015 Mon Sep 17 00:00:00 2001
From: Lionel Debieve <lionel.debieve@foss.st.com>
Date: Fri, 13 Jan 2023 13:57:00 +0100
Subject: [PATCH 0115/1695] plat-stm32mp1: conf: increase MMAP regions number

If EXTERNAL_DT is enabled, the number of MMAP regions must be increased
to avoid issue:

E/TC:0 0 Panic 'Failed to remove temporary Device Tree mapping'
at core/arch/arm/kernel/boot.c:656 <release_external_dt>

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 911a71c85308baa57531d14d625541a0b8bff975 plat-stm32mp1: conf: increase MMAP regions number]
Change-Id: I289fa84f037825262df6d31cb5cd833fa35651b2
Signed-off-by: Lionel Debieve <lionel.debieve@foss.st.com>
---
 core/arch/arm/plat-stm32mp1/conf.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/arch/arm/plat-stm32mp1/conf.mk b/core/arch/arm/plat-stm32mp1/conf.mk
index 98b0207b3..5abc8bb4f 100644
--- a/core/arch/arm/plat-stm32mp1/conf.mk
+++ b/core/arch/arm/plat-stm32mp1/conf.mk
@@ -145,7 +145,7 @@ ifeq ($(CFG_WITH_PAGER),y)
 CFG_WITH_LPAE ?= n
 endif
 CFG_WITH_LPAE ?= y
-CFG_MMAP_REGIONS ?= 23
+CFG_MMAP_REGIONS ?= 30
 CFG_DTB_MAX_SIZE ?= (256 * 1024)
 CFG_CORE_ASLR ?= n
 
-- 
2.47.2

